src/client.js

```diff
@@ -1936,8 +1936,14 @@ function SugarApi(args) {
                     // this is not our message, ignore it
                     return;
                 }
-                $(window).on('message', null);
+
                 var authData = $.parseJSON(event.originalEvent.data);
+                if (!_.isUndefined(authData.guid) && authData.guid.startsWith('pendo-')) {
+                    // ignore pendo message
```

Comment:
pendo message data looks like this:
{"apiKey":"1dd345e9-b638-4bd2-7bfb-147a937d4728","frameId":"Grm39NKI2cc5aF9YZWkJyvLmJPgUOBv6","type":"frame:load","guid":"pendo-1"}

Reply:
I don't think we can do this here, since Ventana is an open source library that can be used by anyone, right? There should not be any dependency on Pendo, or knowledge of Pendo, by Ventana.

Reply:
I agree. This has been modified by checking a new flag for external login in the new PR https://github.com/sugarcrm/ventana/pull/50